istioctl/pkg/admin/istiodconfig.go

```diff
@@ -473,6 +474,9 @@ func istiodLogCmd(ctx cli.Context) *cobra.Command {
 				logCmd.Println(logCmd.UsageString())
 				return fmt.Errorf("--stack-trace-level cannot be combined with --reset, --stack-trace-reset")
 			}
+			if err := util.ValidatePort(controlzPort); err != nil {
```

Comment:
We would need similar changes in he following file as well - https://github.com/istio/istio/blob/master/istioctl/pkg/describe/describe.go